Dockerfile.fuseki

```diff
@@ -0,0 +1,4 @@
+FROM stain/jena-fuseki
+
+RUN apt-get update; \
```

Comment:
Why do we need to compile a Fuseki image? 
We would need to have a very good reason to do that. Normally the official image shall do.